erpnext/accounts/doctype/purchase_invoice/purchase_invoice.json

```diff
@@ -4726,7 +4758,7 @@
  "istable": 0, 
  "max_attachments": 0, 
  "menu_index": 0, 
- "modified": "2019-01-07 16:51:59.800081", 
+ "modified": "2018-12-06 09:00:41.508642", 
```

Comment:
Please use a newer modified timestamp to ensure that the changes you made are updated upon migration.

Reply:
Sorry I saw 2018 instead of 2019...
Just corrected it.
Thanks for a lot for the feedback!